### ISSUE TITLE
My Home -> Mobile App Card: Show a login link if the user has already downloaded the app

### ISSUE THREAD

**Issue by roo2** [Automattic/wp-calypso#55141]
See discussion in https://github.com/Automattic/wp-calypso/issues/54441#issuecomment-887701912 

If the user has ever logged in to the app, show this card instead of the default mobile call to action.
<img width="367" alt="126821839-cdb5d756-8629-4efa-81f8-3fd3b56a445e" src="https://user-images.githubusercontent.com/22446385/128320077-3ce40c92-03d3-45db-a8d0-8349aa97fee3.png">

We can use the `mobile_app_last_seen` user option to determine if a user has ever loaded the mobile app

Clicking on this link would open the app and log the user in if they have logged out.

We need to find out if the app supports this already or if changes will need to be made on the app side as well as in calypso.

**Comment by roo2** [comment]
@mattmiklic, @yaelirub, I made this card last year to implement an improvement suggested by Matt in https://github.com/Automattic/wp-calypso/issues/54441#issuecomment-885804596 But it fell through the cracks as we changed pods.

Do you think that this is worth doing? Team Ganon has some new dev's starting next week who might be able to pick this up.

**Comment by yaelirub** [comment]
Yes we'd still like to update it. @jrushfinn and I were discussing some ideas like different ideas / tests we want to do on the card update like emphasizing the "send link" or adding a QR

**Comment by jrushfinn** [comment]
@roo2 @yaelirub I have put together some ideas here on a mural board: https://app.mural.co/invitation/mural/a8c1359/1642775570168?sender=uc06bf2e30f7339ff45c45613&key=eb26f6c1-dbaf-44b5-a1f4-6768e239afb8
I'm not a designer, so it's not picture-perfect, but I believe this is a great next step that we can test. I'd love feedback on if you think this is all doable.

**Comment by mattmiklic** [comment]
I like the more visual header, rotating headline/subhead, and QR code.

Re: linking the app store buttons to apps.wordpress.com, maybe http://apps.wordpress.com/get/ would be better as it would take the user directly to the app store if they're on iOS or Android?

**Comment by jrushfinn** [comment]
@mattmiklic Thanks for taking a look.

> Re: linking the app store buttons to apps.wordpress.com, maybe http://apps.wordpress.com/get/ would be better as it would take the user directly to the app store if they're on iOS or Android?

Sorry, yes that's the URL I meant to put! I was also envisioning this as a desktop-only design. It looks like this is the banner on mobile web and it does seem to link directly to the app store.
![IMG_1175](https://user-images.githubusercontent.com/88454344/151226834-6fb068a2-8729-4374-a8fc-b5e1e3de31b5.jpg)

**Comment by mattmiklic** [comment]
Ahh right! That makes sense to me.

**Comment by jrushfinn** [comment]
Hi @roo2 , I wanted to follow up here and see if the proposed changes in the comments above work for you. Let me know how else I can help here!

**Comment by autumnfjeld** [comment]
It sounds like two pieces of inter-related work are being discussed here.   
1. Updating the existing "Get the WordPress app" content --> which is issue #55188 .  This is mostly cosmetic, updating content.

2. Changing the content once we know user has downloaded the app, to show more relevant messaging, which is this issue. This is a bit more complex requiring logic to determine "has user already downloaded the app", and then implementing the content as suggested in comments above and in the mural sketch.   


I like the idea of rotating content, with reminders of why the mobile app is useful.



![image](https://user-images.githubusercontent.com/5665959/152279424-263f75ae-4f05-49d8-8626-5b6a61f661be.png)



Seems wise to start with #55188, like adding the QR code (very cool idea!!!!)

**Comment by jrushfinn** [comment]
@autumnfjeld Awesome! Once we have exhausted the more simple conversion optimizations we can explore more dynamic content. No need to overcomplicate initially :)

**Comment by autumnfjeld** [comment]
@jrushfinn Is this issue still applicable?  Or should it be closed?
